ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.14__periodics.yaml

```diff
@@ -131,6 +131,14 @@ tests:
     env:
       CNV_PRERELEASE_VERSION: "4.14"
     workflow: hypershift-kubevirt-conformance
+- as: e2e-kubevirt-mce-conformance
```

Comment:
syncing this test from 4.13.  somehow it got dropped.

Reply:
yep. we'll update some of the env vars later